journey/CONCEPT.md

```diff
@@ -0,0 +1,64 @@
+<p align="center">
+  <a href="https://github.com/ForgeRock/ping-android-sdk">
+    <img src="https://www.pingidentity.com/content/dam/picr/nav/Ping-Logo-2.svg" alt="Ping Identity Logo" width="200">
+  </a>
+  <hr/>
+</p>
+
+# Journey Module Concept
+
+This document describes the plugins for the Journey Module, which integrate
+with  [Orchestrate](https://github.com/ForgeRock/ping-android-sdk/tree/develop/foundation/orchestrate) Module .
+
+## Overview
+
+The Journey Module uses orchestrate module to customize and extend its functionality. Here's an overview of the
+key plugins:
+
+* **CustomerHeader Module:** Injects required headers into Journey API requests.
```

Comment:
Is that a typo? - CustomerHeader -> CustomHeader

Reply:
Updated